src/pages/groupComparison/comparisonGroupManager/ComparisonGroupManager.tsx

```diff
@@ -73,6 +73,27 @@ export default class ComparisonGroupManager extends React.Component<IComparisonG
         this.addSamplesTargetGroupId = "";
     }
 
+    @autobind
+    @action
+    private deleteGroup(group: StudyViewComparisonGroup) {
+        if (this.props.store.isComparisonGroupSelected(group.uid)) {
+            this.props.store.toggleComparisonGroupMarkedForDeletion(group.uid);
+        }
+    }
+
+    @autobind
+    @action
+    private async addSamplesToGroup(group: StudyViewComparisonGroup) {
+        if (this.props.store.selectedSamples.result) {
+            await comparisonClient.updateGroup(
+                group.uid,
+                addSamplesParameters(group, this.props.store.selectedSamples.result)
+            );
+            this.props.store.notifyComparisonGroupsChange();
+            this.cancelAddGroup();
```

Comment:
remove this function call on line 93. It refers to the old UI where it would close the panel.